src/components/ai-input.tsx

```diff
@@ -29,12 +29,42 @@ export const AIInput = ( {
 }: AIInputProps ) => {
 	const inputRef = useRef< HTMLTextAreaElement >( null );
 
+	const [ isTyping, setIsTyping ] = useState( false );
+	const [ typingTimeout, setTypingTimeout ] = useState< NodeJS.Timeout | null >( null );
+
+	const { RiveComponent, inactiveInput, thinkingInput, typingInput, startStateMachine } =
+		useAiIcon();
+
 	useEffect( () => {
 		if ( ! disabled && inputRef.current ) {
 			inputRef.current.focus();
 		}
 	}, [ disabled ] );
 
+	useEffect( () => {
+		startStateMachine();
```

Comment:
Do we need to start the state machine on updates or only once when the component is mounted?

Reply:
Once when mounted should be okay. Addressed in [ee5713](https://github.com/Automattic/studio/pull/280/commits/eee5713e485601ff584fabdf0a1a08ea49c3c2de)

Reply:
Actually... we do need to start the state machine on updates in case it was paused.